it/controllers/ETMPControllerISpec.scala

```diff
@@ -92,6 +92,130 @@ class ETMPControllerISpec extends IntegrationSpecCommonBase with ETMPWiremock {
       |}
       |""".stripMargin)
 
+  val etmpPayloadWithMultipleLSP_InSameCalenderMonth: JsValue = Json.parse(
+    """
+        {
+      |	"pointsTotal": 1,
+      |	"lateSubmissions": 1,
+      |	"adjustmentPointsTotal": 1,
+      |	"fixedPenaltyAmount": 200,
+      |	"penaltyAmountsTotal": 400.00,
+      |	"penaltyPointsThreshold": 4,
+      |	"penaltyPoints": [
+      |		{
+      |			"type": "financial",
+      |			"number": "2",
+      |     "id": "1235",
+      |			"dateCreated": "2021-04-23T18:25:43.511",
+      |			"dateExpired": "2021-04-23T18:25:43.511",
+      |			"status": "ADDED",
+      |			"communications": [
+      |				{
+      |					"type": "secureMessage",
+      |					"dateSent": "2021-04-23T18:25:43.511",
+      |					"documentId": "1234567890"
+      |				}
+      |			]
+      |		},
+      |		{
+      |			"type": "financial",
+      |			"number": "1",
+      |     "id": "1234",
+      |			"dateCreated": "2021-01-01T18:25:43.511",
+      |			"dateExpired": "2021-01-01T18:25:43.511",
+      |			"status": "ACTIVE",
+      |			"period": [{
+      |				"startDate": "2021-01-01T18:25:43.511",
+      |				"endDate": "2021-01-15T18:25:43.511",
+      |				"submission": {
+      |					"dueDate": "2021-05-07T18:25:43.511",
+      |					"submittedDate": "2021-05-12T18:25:43.511",
+      |					"status": "SUBMITTED"
+      |				}
+      |			},
+      |   {
+      |				"startDate": "2021-01-16T18:25:43.511",
+      |				"endDate": "2021-01-31T18:25:43.511",
+      |				"submission": {
+      |					"dueDate": "2021-05-23T18:25:43.511",
+      |					"submittedDate": "2021-05-25T18:25:43.511",
+      |					"status": "SUBMITTED"
+      |				}
+      |			}],
+      |			"communications": [
+      |				{
+      |					"type": "letter",
+      |					"dateSent": "2021-04-23T18:25:43.511",
+      |					"documentId": "1234567890"
+      |				}
+      |			]
+      |		}
+      |	]
+      |}
+      |""".stripMargin)
+
+  val etmpPayloadWithMultipleLSPP_InSameCalenderMonth: JsValue = Json.parse(
```

Comment:
Could you rename this to:
```suggestion
  val etmpPayloadWithMultipleLSPPInSameCalenderMonth: JsValue = Json.parse(
```